content/cli/workspaces.md

```diff
@@ -0,0 +1,207 @@
+
+If we run:
+```bash
+nest new my-project
+```
+
+We've constructed a *standard mode* structure, with a folder structure that looks like this:
+
+<div class="file-tree">
+  <div class="item">src</div>
+  <div class="children">
+    <div class="item">app.controller.ts</div>
+    <div class="item">app.service.ts</div>
+    <div class="item">app.module.ts</div>
+    <div class="item">main.ts</div>
+  </div>
+  <div class="item">node_modules</div>
+  <div class="item">nest-cli.json</div>
+  <div class="item">package.json</div>
+  <div class="item">tsconfig.json</div>
+  <div class="item">tslint.json</div>
+</div>
+
+The `nest-cli.json` file looks like this:
+```javascript
+{
+  "collection": "@nestjs/schematics",
+  "sourceRoot": "src"
+}
+```
+
+We can convert this to a monorepo mode structure as follows:
+
+```bash
+cd my-project
+nest generate app my-app
+```
+
+At this point, `nest` converts the existing structure to a **monorepo mode** structure.  This results in a few important changes.  The folder structure now looks like this:
+
+<div class="file-tree">
+  <div class="item">apps</div>
+    <div class="children">
+      <div class="item">my-app</div>
+      <div class="children">
+        <div class="item">src</div>
+        <div class="children">
+          <div class="item">app.controller.ts</div>
+          <div class="item">app.service.ts</div>
+          <div class="item">app.module.ts</div>
+          <div class="item">main.ts</div>
+        </div>
+        <div class="item">tsconfig.app.json</div>
+      </div>
+      <div class="item">my-project</div>
+      <div class="children">
+        <div class="item">src</div>
+        <div class="children">
+          <div class="item">app.controller.ts</div>
+          <div class="item">app.service.ts</div>
+          <div class="item">app.module.ts</div>
+          <div class="item">main.ts</div>
+        </div>
+        <div class="item">tsconfig.app.json</div>
+      </div>
+    </div>
+  <div class="item">nest-cli.json</div>
+  <div class="item">package.json</div>
+  <div class="item">tsconfig.json</div>
+  <div class="item">tslint.json</div>
+</div>
+
+The `nest` schematic has reorganized the code - moving each **application** project under the `apps` folder, and adding a project-specific `tsconfig.app.json` file.  Our original `my-project` app has become the **default project** for the monorepo, and is now a peer with the just-added `my-app`, located under the `apps` folder.
+
+The `nest-cli.json` file contains details describing the workspace.  It now looks like this:
+
+```javascript
```

Comment:
We should probably mention that the understanding of this file content is not very(!) important

Reply:
Based on this, and your edit removing the `nest-cli.json` file, I decided to move things around a bit.  I think it better addresses the need to show the right level of detail on the `nest-cli.json` file, but puts it at the end so it doesn't interfere with the first-time reader getting through the main concepts.  Let me know what you think.